docs/release-process.md

```diff
@@ -448,28 +426,26 @@ branch:
 
  3. Help the new release managers with their first steps.
 
-## New Release Manager checklist
+## New Release Manager Checklist
 
- 1. Email systems@ to get setup for access to downloads.php.net and to be added
+ 1. Email systems@php.net to get setup for access to downloads.php.net and to be added
     to the release-managers@ distribution list.
-
- 2. Create a GPG key for your @php.net address and publish it by editing
+    
+ 2. Request membership for the following repositories:
```

Comment:
This can be simplified to just asking for membership in the "Release Managers" team on github. That team provides access to the repos listed below.